scripts/deploy-ebook-to-www

```diff
@@ -99,10 +153,13 @@ do
 	convert -resize "196" -sampling-factor 4:2:0 -strip -quality 75 -colorspace RGB -interlace JPEG "${imgWorkDir}/${urlSafeIdentifier}.png" "${imgWorkDir}/${urlSafeIdentifier}.jpg"
 	convert -resize "392" -sampling-factor 4:2:0 -strip -quality 75 -colorspace RGB -interlace JPEG "${imgWorkDir}/${urlSafeIdentifier}@2x.png" "${imgWorkDir}/${urlSafeIdentifier}@2x.jpg"
 
-	mv "${imgWorkDir}/${urlSafeIdentifier}@2x.jpg" "${imgWorkDir}/${urlSafeIdentifier}-cover@2x.jpg"
-	mv "${imgWorkDir}/${urlSafeIdentifier}.jpg" "${imgWorkDir}/${urlSafeIdentifier}-cover.jpg"
+	hash="$(git log -n 1 --pretty=format:"%ct %s" | sha1sum | awk '{ print substr ($1,1,8) }')"
```

Comment:
In this block, it's not necessary to do anything different from the original. The web server accepts requests for the hero that include a unique hash, so that the images aren't cached if they're changed. Apache then rewrites those requests to remove the hash and serve the image generated by this script instead.

Reply:
Oh, I see. That means I'll have to change the Vagrant setup instead.